packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/Utils.java

```diff
@@ -83,6 +83,10 @@ public static FileParams getFileUploadParams(File file) throws PangeaException {
 		return new FileParams(size, sha256, crc);
 	}
 
+	public static long getFileSize(File file) throws PangeaException {
```

Comment:
I don't think this can actually throw a `PangeaException`?

Reply:
Fixed!